src/app/legacy/containers/ConsentBanner/Banner/cookie.amp.jsx

```diff
@@ -160,8 +159,58 @@ const OptionsItem = styled.li`
   button[on='tap:AMP.setState({ isManagingSettings: true })'] {
     background: none;
     border: ${BORDER_WIDTH} solid ${props => props.theme.palette.CONSENT_ACTION};
+    border: ${BORDER_WIDTH} solid ${props => props.theme.palette.CONSENT_ACTION};
     color: ${props => props.theme.palette.CONSENT_ACTION};
   }
+
+  &.hide {
+    width: 44px;
+    height: 44px;
+    position: absolute;
+    top: 0;
+    right: 0;
+    margin: 0;
+    padding: 0;
+    & button {
+      width: 2.75rem;
+      height: 2.75rem;
+      cursor: pointer;
+      background: none;
+      border: none;
+      &:focus::after,
+      &:hover::after {
+        ${a11yOutlinePosition}
+        border: ${focusIndicatorThickness} solid
+          ${props => props.theme.palette.WHITE};
+      }
+      &:focus-visible::after {
+        ${a11yOutlinePosition}
+        border: ${focusIndicatorThickness} solid
+          ${props => props.theme.palette.BLACK};
+        box-shadow: 0 0 0 ${focusIndicatorThickness}
+          ${props => props.theme.palette.WHITE} inset;
+      }
+    }
+    & svg {
+      color: white;
+      fill: currentColor;
+      position: absolute;
+      top: 0;
+      right: 0;
+    }
+  }
+`;
+
+const ContentWrapper = styled.div`
+  align-self: center;
+  max-height: ${BANNER_MAX_HEIGHT};
+  max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX};
+  overflow-y: auto;
+  padding: 2.75rem ${GEL_SPACING_DBL} ${GEL_SPACING} ${GEL_SPACING_DBL};
```

Comment:
I think this can be removed as it looks like its adding a bit too much padding.

Reply:
Thanks for the link! I wonder if thats just for the mobile breakpoint to have that padding set? I'm happy for it to go in as is though and we can check with UX later. The vast majority of AMP users will be mobile anyway, so its all good :) 

Good spot though, we will need to keep that padding in for mobile at the very least!